internal/app/handle.go

```diff
@@ -128,3 +170,41 @@ func handleGokuDefault(w http.ResponseWriter, r *http.Request) {
 	w.Write(d)
 	logger.Info().Msg("Message written\n")
 }
+
+// the elasticClient function creates an elastic search client and
+// adds an index named fighters that will use the mapping variable to set
+// the layout of its body if it doesn't already exist.
+func elasticClient() (context.Context, *elastic.Client) {
```

Comment:
if you want to reuse the context struct here, you should pass a `*context.Context` as a parameter, instead of returning a `new context.Context`